src/services/ltiTool/index.js

```diff
@@ -21,6 +23,24 @@ module.exports = function () {
 	const ltiToolService = app.service('/ltiTools');
 	ltiToolService.hooks(hooks);
 
+	app.use('/tools/:id/link', {
+		create (data, params) {
+			return app.service('ltiTools').find({
+				query: {
+					oAuthClientId: jwt.decode(data.id_token).iss,
+					lti_version: '1.3.0',
+					isTemplate: true,
+				},
+			}).then((tool) => {
+				const idToken = jwt.verify(data.id_token, tool.data[0].key, {algorithm: 'RS256'});
+				const link = idToken['https://purl.imsglobal.org/spec/lti-dl/claim/content_items'];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['link' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4846475)